opal/mca/event/libevent2021/libevent2021_component.c

```diff
@@ -120,12 +120,14 @@ const opal_event_component_t mca_event_libevent2021_component = {
     }
 };
 
+#define BUFF_SIZE 1024
```

Comment:
Are you happy that using BUFSIZ on some platforms this might now be as small as 256?